backend/battle/forms.py

```diff
@@ -2,17 +2,22 @@
 from django.contrib.auth.forms import UserCreationForm
 
 from battle.models import Battle, PokemonTeam, Team
-from battle.battles.battle import validate_sum_pokemons, verify_pokemon_is_saved
+from battle.battles.battle import save_pokemons_if_they_doenst_exist
 
 from users.models import User
 
 from pokemon.models import Pokemon
 
-from pokemon.helpers import verify_pokemon_exists_api
-
-from services.create_battle import (
+from services.battle import (
     validate_if_creator_and_opponent_has_different_contenders,
-    fetch_opponent_or_create_if_doenst_exist, send_invite_email_or_create_password_email)
+    fetch_opponent_or_create_if_doenst_exist,
+    send_invite_email_or_create_password_email,
+    verify_pokemons_fields_has_missing_values,
+    verify_fields_has_wrong_pokemon_name,
+    verify_positions_fields_has_missing_values,
+    verify_pokemon_sum_is_valid,
+    verify_positions_fields_has_duplicate_values,
```

Comment:
I think all these helpers that start with `verify` should start with `has` or `is`

Reply:
hmm okay! I misunderstood, thanks!